src/dg/dg.cpp

```diff
@@ -869,9 +1012,9 @@ void DGBase<dim,real,MeshType>::assemble_cell_residual (
             const real penalty = 0.5 * (penalty1 + penalty2);
 
             const dealii::types::global_dof_index neighbor_cell_index = neighbor_cell->active_cell_index();
-            //if ( compute_dRdW || compute_dRdX || compute_d2R ) {
```

Comment:
If this is to be removed, untab the code below for consistency

Reply:
fixed